functional_tests/test_categories.py

```diff
@@ -52,3 +52,13 @@ def testGetCategoryByName(self):
         response = discourse_client.categories.getByName('Broncos').get_result()
 
         self.assertEqual('Broncos', response['name'])
+
+    def testGetAllCategories(self):
+        discourse_client.categories.create(
```

Comment:
Could we create a couple of categories, just to make sure it returns all of them?